.eslintignore

```diff
@@ -32,8 +32,8 @@ tools/http-helpers.js
 tools/inspector.js
 tools/index.js
 tools/mongo-exit-codes.ts
-tools/processes.ts
-tools/progress.js
+tools/processes.js
```

Comment:
@benjamn - I think this should stay `.ts`, probably just a weird merge conflict resolution after merging the other PR